odl/util/testutils.py

```diff
@@ -298,11 +298,12 @@ class FailCounter(object):
     ``*** FAILED 1 TEST CASE(S) ***``
     """
 
-    def __init__(self, test_name, err_msg=None):
+    def __init__(self, test_name, err_msg=None, quiet=False):
```

Comment:
That's true. When I started I had the mindset of totally silencing the counter if desired, but that's not what it ended up to be.